Sentinel/Classes/Core/PreferencesInfo/Items/ToggleToolItem.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ToggleToolItem.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 17.03.2025..
+//
+
+import Foundation
+
+/// Item which will create a preferences view with a toggle button
+public struct ToggleToolItem: PreferenceItem {
```

Comment:
Every other item has a prefix `Preference`, so I would recommend going with something like `PreferenceToggleItem`.

You can add a typealias `ToggleToolItem` for `PreferenceToggleItem` and add a deprecation message for it, to still support that type from the "outside".

Reply:
Great idea, I would also like to add the typealias so that we don't introduce breaking changes in the next release.